app/src/main/java/org/mozilla/fenix/perf/StartupFrameworkStartMeasurement.kt

```diff
@@ -19,13 +15,9 @@ import org.mozilla.fenix.GleanMetrics.StartupTimeline as Telemetry
  * Timespan metric start/stop programming model so it lives in its own class.
  */
 internal class StartupFrameworkStartMeasurement(
-    private val stat: Stat = Stat(),
```

Comment:
chore (non-blocking): afaict, the Stat class is now unused: can we remove it and its test?

Reply:
Stat classes has been removed